source/manual/ab-testing.html.md

```diff
@@ -13,13 +13,15 @@ For a general introduction to A/B testing from a content design perspective, see
 
 [cdn]: https://docs.publishing.service.gov.uk/manual/cdn.html
 
-![](https://docs.google.com/drawings/d/e/2PACX-1vR67bcDfNDaa4buyKGKQev0xUsjcD9RzjNCjWGhr0HJtXRmSNaltPJotXjwmKUmZj0ZH_B2xAymEYbV/pub?w=1330&h=517)
+> A/B tests are only enabled for users who have opted in to analytics cookies.
 
-Source: [GOV.UK Architecture Google Drive](https://docs.google.com/drawings/d/1rx4brKZBzj-9q3evkiUw2MbqwqTYWkc0Lku6u3cLXqU/edit)
+If the user is opted-in to usage cookies, but hasn't got an AB test cookie yet:
 
-## How A/B testing works
+![](images/ab-testing/opted-in.svg)
```

Comment:
I can't see the images in the preview view.

I think that also, a brief description of what the image is depicting might be useful. The explanation can be nabbed from [this comment](https://github.com/alphagov/govuk-cdn-config/pull/462#issuecomment-1614841478).